sdk/resourcemanager/azure-resourcemanager-network/src/test/java/com/azure/resourcemanager/network/NetworkInterfaceOperationsTests.java

```diff
@@ -463,6 +508,30 @@ public void canListSubnetAvailableIpAddresses() {
 
         availableIps = subnet.listAvailablePrivateIPAddresses();
         Assertions.assertFalse(availableIps.contains(availableIp));
+
+        // address prefixes
+        network.update().defineSubnet(subnet2Name)
+            .withAddressPrefixes(Arrays.asList("10.0.0.8/29"))
```

Comment:
We probably want to do some test with more than 1 addressPrefixes.

Reply:
Yeah, let me add one.